core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -50,6 +50,51 @@ gdb_bootargs:
 	.word 0
 #endif
 
+#ifdef CFG_PL310
+.section .rodata.init
+panic_boot_file:
+	.asciz __FILE__
+
+/*
+ * void assert_flat_mapped_range(uint32_t vaddr, uint32_t line)
+ */
+.section .text.init
+LOCAL_FUNC __assert_flat_mapped_range , :
+UNWIND(	.fnstart)
+UNWIND(	.cantunwind)
+	push	{ r4-r5, lr }
```

Comment:
Please push an even number of registers to maintain required alignment of the stack pointer.